app/controllers/home_controller.rb

```diff
@@ -1,12 +1,12 @@
 class HomeController < ApplicationController
 
   def index
-  @chapters = Chapter.all
-  @chapter = Chapter.all
+  @chapters = Chapter.where("is_active = 1")
```

Comment:
Gladly! Really appreciate these pointers, @kexline4710 -- thanks! That'll be way cleaner...